src/main/java/org/folio/edge/sip2/MainVerticle.java

```diff
@@ -60,9 +67,15 @@ public MainVerticle(Map<Command, ISip2RequestHandler> handlers) {
     this.handlers = handlers;
   }
 
+  private String getSanitizedConfig() {
+    JsonObject sc = config().copy();
```

Comment:
This code reads both the config file stored in local storage and on S3?

Reply:
These highlighted lines just make a copy of the config that was loaded from local file. The copy is needed to allow blocking out any security credentials in the config before printing to the logs.  
The MainVerticle does now read both the config file stored in local storage and on S3; the S3 config is loaded by the ConfigRetreiver while the MainVerticle is starting up, just before it starts listening for connections.